mango/plots/plots.py

```diff
@@ -41,8 +44,30 @@ def plotly_plot_lines(
         df, x=x_axis, y=y_axis, hover_data=hover_list, color=color_by, **kwargs_plotly
     )
 
-    # add title
-    fig.update_layout(title_text=title)
+    position_dict = {"topright": {"xanchor": "right", "yanchor": "top"},
+                      "topleft": {"xanchor": "left", "yanchor": "top"},
+                      "bottomright": {"xanchor":"right", "yanchor":"bottom"},
+                      "bottomleft": {"xanchor":"left", "yanchor":"bottom"}}
+
+    if legend and position is not None:
+        try:
```

Comment:
No need for try except. Even if used, except should only catch KeyError not all exceptions.

```python
assert position in translate_args.keys(), f"{position} must be a one of the following {list(translate_args.keys())}"
fig.update_layout(...)
```

I prefer raising an error rather than looking like it works but legend placed incorrectly.